docs/features/startup_commands.rst

```diff
@@ -1,2 +1,66 @@
 Startup Commands
 ================
+
+``cmd2`` provides a couple different ways for running commands immediately
+after your application starts up:
+
+1. Commands at Invocation
+1. Startup Script
+
+Commands run as part of a startup script are always run immediately after the
+application finishes initializing so they are guaranteed to run before any
+*Commands At Invocation*.
+
+
+Commands At Invocation
+----------------------
+
+.. _Argparse: https://docs.python.org/3/library/argparse.html
+
+You can send commands to your app as you invoke it by including them as extra
+arguments to the program. ``cmd2`` interprets each argument as a separate
+command, so you should enclose each command in quotation marks if it is more
+than a one-word command.  You can use either single or double quotes for this
+purpose.
+
+.. code-block:: shell
+
+  $ python examples/example.py "say hello" "say Gracie" quit
+  hello
+  Gracie
+
+You can end your commands with a **quit** command so that your ``cmd2``
+application runs like a non-interactive command-line utility (CLU).  This
+means that it can then be scripted from an external application and easily used
+in automation.
+
+.. note::
+
+   If you wish to disable cmd2's consumption of command-line arguments, you can
+   do so by setting the  ``allow_cli_args`` argument of your ``cmd2.Cmd`` class
+   instance to ``False``.  This would be useful, for example, if you wish to
+   use something like Argparse_ to parse the overall command line arguments for
```

Comment:
What's causing the underline and extra space?

Reply:
That is reStructuredText formatting.  The underscore at the end means it is inserting a link defined elsewhere.  The extra indentation is for grouping what is in the Note.